www/outsystems-secure-sqlite-init.js

```diff
@@ -38,6 +62,7 @@ function acquireLsk(successCallback, errorCallback) {
                     function (value) {
                         lskCache = value;
                         console.log("Got Local Storage key");
+                        OutSystemsNative.Logger.logWarning("Got Local Storage key (" + hashCode(lskCache) + ")", "SecureSQLiteBundle");
```

Comment:
The hash function crashes if the `string` is `null`, log `"null"` in that case.

Reply:
Done :)